doc/gitian-building.md

```diff
@@ -1,479 +1,159 @@
+--docker            Use Docker instead of KVM
+-o|--os lwx         Specify which Operating Systems the build is for. Default is lwx,
+                    l for Linux, w for Windows, x for MacOS
+-j proc             Number of processes to use. Default 2
+-m n                Memory to allocate in MiB. Default 2000
+--enable-cache      Use local apt-cacher server. If you need to specify host, use
+                    MIRROR_HOST environment variable
+
+-c|--commit         Indicate that the version argument is for a commit or branch
+-u|--url repo       Specify the URL of the repository. Default is https://github.com/dogecoin/dogecoin
+--test              CI TEST. Uses Docker
+-h|--help           Print this help message
+```
+
+### Example
+
+The entire gitian flow can be performed step by step, example using docker :
 ```bash
-URL=https://github.com/laanwj/dogecoin.git
-COMMIT=2014_03_windows_unicode_path
-./bin/gbuild --commit dogecoin=${COMMIT} --url dogecoin=${URL} ../dogecoin/contrib/gitian-descriptors/gitian-linux.yml
-./bin/gbuild --commit dogecoin=${COMMIT} --url dogecoin=${URL} ../dogecoin/contrib/gitian-descriptors/gitian-win.yml
-./bin/gbuild --commit dogecoin=${COMMIT} --url dogecoin=${URL} ../dogecoin/contrib/gitian-descriptors/gitian-osx.yml
-```
-
-Building fully offline
------------------------
-
-For building fully offline including attaching signatures to unsigned builds, the detached-sigs repository
-and the dogecoin git repository with the desired tag must both be available locally, and then gbuild must be
-told where to find them. It also requires an apt-cacher-ng which is fully-populated but set to offline mode, or
-manually disabling gitian-builder's use of apt-get to update the VM build environment.
-
-To configure apt-cacher-ng as an offline cacher, you will need to first populate its cache with the relevant
-files. You must additionally patch target-bin/bootstrap-fixup to set its apt sources to something other than
-plain archive.ubuntu.com: us.archive.ubuntu.com works.
+# Download Gitian dependencies
+$ ./gitian-build.sh --docker --setup 1.14.4
 
-So, if you use LXC:
-
-```bash
-export PATH="$PATH":/path/to/gitian-builder/libexec
-export USE_LXC=1
-cd /path/to/gitian-builder
-./libexec/make-clean-vm --suite trusty --arch amd64
-
-LXC_ARCH=amd64 LXC_SUITE=trusty on-target -u root apt-get update
-LXC_ARCH=amd64 LXC_SUITE=trusty on-target -u root \
-  -e DEBIAN_FRONTEND=noninteractive apt-get --no-install-recommends -y install \
-  $( sed -ne '/^packages:/,/[^-] .*/ {/^- .*/{s/"//g;s/- //;p}}' ../dogecoin/contrib/gitian-descriptors/*|sort|uniq )
-LXC_ARCH=amd64 LXC_SUITE=trusty on-target -u root apt-get -q -y purge grub
-LXC_ARCH=amd64 LXC_SUITE=trusty on-target -u root -e DEBIAN_FRONTEND=noninteractive apt-get -y dist-upgrade
-```
-
-And then set offline mode for apt-cacher-ng:
+# Build & sign executables
+$ ./gitian-build.sh --docker --build --sign SIGNER 1.14.4
 
+# Verify signatures
+$ ./gitian-build.sh --verify 1.14.4
```

Comment:
Matter of choice, but I think comments are more obvious as it is, by having comments at the previous line rather than having it at the end of the line. I would prefer how it is currently.

Is it ok for you to keep it as it is, or do you think this change worth it ?